SourceWriter/SourceWriter/Main.storyboard

```diff
@@ -1178,9 +1178,9 @@ Gw
                                         <action selector="OriginalSize:" target="n8A-zD-eLP" id="jHe-Df-YoN"/>
                                     </connections>
                                 </toolbarItem>
-                                <toolbarItem implicitItemIdentifier="NSToolbarPrintItem" explicitItemIdentifier="Print" id="WFE-hK-GLq" customClass="ManualToolbarItem">
+                                <toolbarItem implicitItemIdentifier="NSToolbarPrintItem" explicitItemIdentifier="PrintPreview" id="WFE-hK-GLq" customClass="ManualToolbarItem">
                                     <connections>
-                                        <action selector="print:" target="n8A-zD-eLP" id="NeO-eg-Fvs"/>
+                                        <action selector="printPreview:" target="n8A-zD-eLP" id="NeO-eg-Fvs"/>
```

Comment:
Why this change needed?

Reply:
Sorry, it was my mistake. Initially I thought that it was the reason of the warning.